core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/codesystem/CodeSystemService.java

```diff
@@ -45,32 +43,9 @@
 @Component
 public final class CodeSystemService {
```

Comment:
CodeSystemRestService and CodeSystemRestService are still sharing the remaining service method. I didn't want to touch this part to avoid expanding the PR's scope.